pom.xml

```diff
@@ -144,7 +144,7 @@
         <mvn.version>3.2.5</mvn.version>
         <project.build.outputTimestamp>2023-10-04T08:38:05Z</project.build.outputTimestamp>
 
-        <jakarta.annotation.version>3.0.0</jakarta.annotation.version>
+        <jakarta.annotation.version>2.1.1</jakarta.annotation.version>
```

Comment:
Why this downgrade?

Reply:
https://github.com/eclipse-ee4j/glassfish-hk2/pull/1065/commits/80d7cd2c9f87f9669cb96c9320ea19a58bb51214